### ISSUE TITLE
Auto suggest internal links from a folder

### ISSUE THREAD

**Issue by ooker777** [tadashi-aikawa/obsidian-various-complements-plugin#154]
Similar to the [internal link complement](https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/1.%20Features/Internal%20link%20complement/ "Internal link complement - Various Complements") feature, but only for notes inside a specific folder. Is it possible?

**Comment by tadashi-aikawa** [comment]
Hi, @ooker777 :)

Maybe I can add the [⚙️Include prefix path patterns] option about Internal link complement, same as Current vault complement. Does it satisfy your requirement?

[⚙️Include prefix path patterns]: https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/4.%20Options/4.5.%20Current%20vault%20complement/%E2%9A%99%EF%B8%8FInclude%20prefix%20path%20patterns/

**Comment by ooker777** [comment]
Cool. Can the source folder be more adapt to the environment? For example if I type `aa` then it will suggest notes from folder A, if I type `bb` it will suggest notes from folder B?

**Comment by tadashi-aikawa** [comment]
> Can the source folder change depending to the environment?

@ooker777 
No. I will not implement it because I think it should use Obsidian's core internal link suggestions.

**Comment by ooker777** [comment]
Ah, so this feature is not on the api yet?

**Comment by tadashi-aikawa** [comment]
> if I type aa then it will suggest notes from folder A, if I type bb it will suggest notes from folder B?

Sorry, I misread your comment. I thought `aa` and `bb` were folder names, but they were environments.

It may be possible to switch environments, but due to the nature of the plug-in, it seems difficult to do so in response to input. Probably, it should uses commands.

**Comment by ooker777** [comment]
What commands?

**Comment by tadashi-aikawa** [comment]
There are no commands now.
